mobile-app/components/BorderCell.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Polygon } from 'react-native-svg';
+import { PLAYER_1_CELL_COLOR, PLAYER_2_CELL_COLOR } from "../utils";
+import { CellType } from "../utils";
+
+interface BorderCellData {
+  svgPoints: string;
+  strokeColor: string;
+  playerBorder: CellType;
+}
+
+
+export default function BorderCell(props: BorderCellData) {
+  return (
+    <Polygon points={props.svgPoints} stroke={props.strokeColor} fill={getBorderColor(props.playerBorder)} />
+  )
+}

```

Comment:
```suggestion
export default function BorderCell({ svgPoints, strokeColor, playerBorder }: BorderCellData) {
  return (
    <Polygon points={svgPoints} stroke={strokeColor} fill={getBorderColor(playerBorder)} />
  )
}
```

Is easier to read IMHO